infra/modules/kubevela-addons/definitions/cardano-node-connector.cue

```diff
@@ -0,0 +1,34 @@
+import "encoding/json"
+
+parameter: {
+  network: *"preview" | string
+  version?: *"latest" | string
+  volumeMounts: *[{
```

Comment:
I see this is still a draft, is the plan to generate the volumeMount stanza from a requested mount location?

Reply:
yep, the intention was to make it as dynamical as possible.

some applications could use a different `node.sock` path instead of just /ipc or maybe even in the case they use their cardano-node instance in a different default path.

we take /ipc as default tho and let the user override those values if needed,

sounds reasonable?

Reply:
Yes, thanks!